apps/frontend/src/components/TopBar.tsx

```diff
@@ -13,12 +13,12 @@ import {
 import { secondsToHoursMinutesAndSecondsString } from '@dundring/utils';
 
 export const TopBar = () => {
-  const { power, cadence } = useSmartTrainer();
+  const { power, cadence, currentResistance } = useSmartTrainer();
   const { heartRate } = useHeartRateMonitor();
-  const { activeWorkout, activeFtp } = useActiveWorkout();
+  const { activeWorkout } = useActiveWorkout();
   const { timeElapsed, distance, speed } = useData();
   const remainingTime = getRemainingTime(activeWorkout);
-  const targetPower = getTargetPower(activeWorkout, activeFtp);
+  const targetPower = currentResistance;
```

Comment:
Maybe we either should have `targetPower` use result from `getTargetPower` as a fallback, in case `currentResistance` is `undefined` or `0`? 🤠 

If not; it might be ok to just use the `currentResistance` variable directly instead of `targetPower`. And we should also remove the unused import of `getTargetPower`

Reply:
`currentResistance` has type`number`, so it shouldnt be `undefined`, right?

I am not 100%, but if it is a mismatch, it feels like a bug? So it would maybe be a bit hard to find if we mask it by some other value.

Reply:
Seems to me that it will always be at least `0`?